open-api/rest-catalog-open-api.yaml

```diff
@@ -1548,6 +1611,34 @@ components:
           items:
             type: string
 
+    ServerCapability:
+      type: object
+      description: Describes a capability with versioning information supported by the server. A server is required to implement all endpoints grouped under a particular capability.
+        If a server only partially implements a given capability, then not implemented endpoints grouped under that tag must throw `501 Not Implemented`.
+      required:
+        - capability
+        - versions
+      properties:
+        capability:
+          type: string
+          description: |
+            A capability supported by the server. The currently available capabilities are:
+             - tables
+             - views
+             - remote-signing
+             - vended-credentials
+             - multi-table-commit
```

Comment:
We discussed in the last sync meeting that the capability is useful when it impacts the client's fallback behavior. In that case, it seems to me that only `views` is a necessary capability. What other capabilities here would impact fallback behavior?